test/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/CheckEoriDetailsControllerSpec.scala

```diff
@@ -112,7 +114,21 @@ class CheckEoriDetailsControllerSpec
       def performAction(data: Seq[(String, String)]): Future[Result] =
         controller.submit()(FakeRequest().withFormUrlEncodedBody(data: _*))
 
-      "The user chooses the yes option" in {
+      "The user chooses the yes option with FeatureSwitch.Bulk enabled" in {
+        featureSwitch.BulkClaim.enable()
+        val (session, fillingOutClaim, _) = sessionWithClaimState()
+
+        inSequence {
+          mockAuthWithNoRetrievals()
+          mockGetSession(session.copy(journeyStatus = Some(fillingOutClaim)))
+        }
+
+        val result = performAction(Seq(CheckEoriDetailsController.dataKey -> "0"))
+        checkIsRedirect(result, routes.SelectNumberOfClaimsController.show(false))
+      }
+
+      "The user chooses the yes option with FeatureSwitch.Bulk disabled" in {
```

Comment:
Change to "Redirects to EnterMovementReferenceNumber if user says details are correct and FeatureSwitch.Bulk is enabled"